node/src/chain_spec.rs

```diff
@@ -15,15 +15,19 @@ use sp_core::crypto::Ss58Codec;
 // Specialized `ChainSpec`. This is a specialization of the general Substrate ChainSpec type.
 pub type ChainSpec = sc_service::GenericChainSpec<GenesisConfig>;
 
+// These functions are unused in production compiles, util functions for unit testing
+#[allow(dead_code)]
```

Comment:
Would it be possible to instead, or in addition, utilize conditional compiler trickery?

```rust
#[cfg(not(release))]
```

Reply:
It'll still throw a compiler warning even when compiling in debug